modular_skyrat/modules/modular_items/lewd_items/code/lewd_quirks.dm

```diff
@@ -77,41 +80,43 @@
 			switch(rand(1,6))
 				if(1)
 					if(stress >= 100)
-						to_chat(owner, span_purple("You feel slightly aroused..."))
+						to_chat(owner, span_purple("Your breath grows hot and heavy..."))
+						owner.adjustOxyLoss(8)
+						owner.emote("exhale")
 					else
-						to_chat(owner, span_purple("Lust spreads over your body!"))
-						owner.emote("moan")
+						to_chat(owner, span_purple("You feel hot and seduced!"))
+						owner.adjustOxyLoss(16)
+						owner.emote("gasp")
+						
 				if(2)
 					if(stress >= 100)
 						to_chat(owner, span_purple("You can't stop shaking..."))
 						owner.do_jitter_animation(20)
+						owner.emote("shiver")
 					else
-						to_chat(owner, span_purple("You feel hot and seduced!"))
+						to_chat(owner, span_purple("Lust spreads over your body!"))
 						owner.dizziness += 20
 						owner.add_confusion(20)
 						owner.Jitter(20)
 						owner.do_jitter_animation(20)
-						owner.adjustStaminaLoss(50)
+						owner.emote("twitch")
+            
 				if(3, 4)
 					if(stress >= 100)
-						to_chat(owner, span_purple("You bring your hips together in lust."))
+						to_chat(owner, span_purple("You bring your hips together in lust..."))
 					else
-						to_chat(owner, span_purple("Desire is driving you mad!"))
-						owner.hallucination += 30
+						to_chat(owner, span_purple("You press your thighs together in lust, stumbling slighty!"))
+						owner.emote("stumble")
+						owner.Immobilize(2 SECONDS, TRUE)
 				if(5)
 					if(stress >= 100)
-						to_chat(owner, span_purple("You feel like your genitals are burning..."))
```

Comment:
This makes more sense than the replacement line

Reply:
that was a hold over from pre-cleanup, good catch